client/src/site-search/search-results.tsx

```diff
@@ -282,98 +285,137 @@ function Results({
   metadata: Metadata;
   suggestions: Suggestion[];
 }) {
+  const hitCount = metadata.total.value;
   const locale = useLocale();
+  const page = metadata["page"];
   const [searchParams] = useSearchParams();
+  const query = searchParams.get("q");
 
   return (
-    <div>
-      <div className="search-results">
-        <p>
-          Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
-          milliseconds.
-        </p>
+    <>
+      {hitCount > 0 ? (
+        <>
+          <h1>
+            Search results {page && page !== 1 && `page ${page} `}for{" "}
```

Comment:
This doesn't read well, when you're on page 2. 
<img width="904" alt="Screen Shot 2021-02-23 at 9 05 18 AM" src="https://user-images.githubusercontent.com/26739/108854941-6db4df80-75b6-11eb-88ac-ba6cbc0ba9b9.png">

The sentence "Search results for *margin*" made sense. It's not spoken English but it made sense.
However, "Search results page 2 for *margin*"  doesn't make sense. 

I think it was better before. Then the "Page 2" part "stood out" so that you know you're still on the same search but just on a different page. 
<img width="593" alt="Screen Shot 2021-02-23 at 9 09 03 AM" src="https://user-images.githubusercontent.com/26739/108855242-cab09580-75b6-11eb-97df-d943243e3579.png">
Having said that, I prefer the "Search results for ..." instead of "Results: ..."

Reply:
What about: "Search results for margin, page 2" - I am just thinking how a screen reader would read it to the end-user.

Reply:
And the "tab" title can perhaps be the same? Often that is the first thing that is read to screen reader users, especially when they tab between tabs or windows.

Reply:
So, something like "Search results for, magic, page 2"

Reply:
That's still not a good sentence. It doesn't read well. "Search results for magic" is fine. "Search results for magic. Page 2" is OK. If you say that out loud, with the relevant pause for the `.` it actually sounds OK.